scopes/preview/preview/strategies/component-strategy.ts

```diff
@@ -365,18 +365,18 @@ export class ComponentBundlingStrategy implements BundlingStrategy {
     const moduleMapsPromise = defs.map(async (previewDef) => {
       const moduleMap = await previewDef.getModuleMap([component]);
       const maybeFiles = moduleMap.get(component);
-      if (!maybeFiles || !capsule) return [];
+      if (!maybeFiles || !capsule) return { prefix: previewDef.prefix, paths: [] };
```

Comment:
still show empty previews in link file (otherwise we could think the preview is missing)